components/Auth/AuthDialog.js

```diff
@@ -0,0 +1,149 @@
+import PropTypes from 'prop-types';
+import Image from 'next/image';
+import {
+  Box,
+  Button,
+  Dialog,
+  Divider,
+  Grid,
+  Hidden,
+  Typography,
+  IconButton,
+} from '@material-ui/core';
+
+import styles from '../../styles/Auth/Auth.module.css';
+
+const AuthDialog = ({
+  modalOpen,
+  setModalOpen,
+  handleCredentialsSubmit,
+  formContent,
+  isSignUp,
+  setIsSignUp,
+  validSignUp,
+  btnText,
+  oAuthText,
+  switchModeText,
+  signIn,
+  providers,
+}) => {
+  return (
+    <Dialog
+      fullWidth={true}
+      maxWidth="lg"
+      open={modalOpen}
+      onClose={() => setModalOpen(false)}
+      aria-labelledby="Login/Signup Modal"
+      className={styles.Dialogue}>
+      <Grid container className="my-8" alignItems="center" direction="row">
```

Comment:
![image](https://user-images.githubusercontent.com/39652186/122316153-585b1c80-cee9-11eb-8eda-e9a6e7826ec9.png)
future task: make responsive, has a vertical scroll on my screen at the moment (prob bc my laptop screen is smaller ;-;)

Reply:
thanks. Didn't take that into account.